src/asmcnc/apps/drywall_cutter_app/widget_drywall_shape_display.py

```diff
@@ -250,11 +246,11 @@ def __init__(self, **kwargs):
         self.dwt_config = kwargs['dwt_config']
         self.kb = kwargs['kb']
 
-        self.d_input.bind(text=self.d_input_change) # Diameter of circle
-        self.l_input.bind(text=self.l_input_change) # Length of line
-        self.r_input.bind(text=self.r_input_change) # Radius of corners
-        self.x_input.bind(text=self.x_input_change) # Square/rectangle x length
-        self.y_input.bind(text=self.y_input_change) # Square/rectangle y length
+        self.d_input.bind(focus=lambda instance, _: self.text_input_change(instance, 'd')) # Diameter of circle
```

Comment:
Do you need to be specifying 'd'/'x'/etc in a lambda? If you just had self.d_input.bind(focus=self.text_input_change), you could just check the instance is d_input/x_input etc?

Reply:
I just saw the text_input_change after scrolling and understand the reason but this way seems a little less transparent. I think implementing the events with the parameters is a neater way which will fit consistency with others.